### ISSUE TITLE
Balance Claims: All Test Cases

### ISSUE THREAD

**Issue by jcalfee** [cryptonomex/graphene-ui#98]
A BTS 2.0 balance claim assigns any number of private keys to a single registered account.  The balance claims should mimic the import key file as much as possible.  The following cases may happen when importing keys from BTS 1.0 into BTS 2.0:
- [x] Multiple private keys assigned to one registered account
- [x] Two balance claims on separate registered accounts with separate private keys
- [x] Private key with no registered account (the account name is an empty string "").  Faucet registration is required  before the balance claim can happen.
- [x] Private key assigned to someone else's account.  Scenario: BTS 1.0 user has a key balance and an unregistered account name already in their wallet, someone else later registered the account (someone registered 20K accounts and happens to grab the name, etc), original user goes back and exports this account and key and tries to import into graphene-ui and claim.
- [x] Private key assigned to two different registered accounts (user needs to choose)
- [x] table should not use center alignment
- [x] Make balances selectable.  Automatically check all matching balances above.
- [x] Make sure the user interface can handle a wallet with 1000+ balances, in other words the top table needs to be scrollable.

In summary, allows the user to assign private keys grouped by there 'labeled' account names into real BTS 2.0 accounts.

**Comment by jcalfee** [comment]
The following shows what multiple accounts might look like.  Note, the multi-selection list also has the effect of not selecting any for the user by default and making them select one first.
![image](https://cloud.githubusercontent.com/assets/204121/9112126/53c6d918-3c12-11e5-88c4-00c0efeaa704.png)

I'm thinking, that the "Claim balance to account" will be filtered.  Only accounts that have private keys in the wallet will be listed (un-registered accounts with a balance above will not be listed).  The title will change to indicate "Registered Accounts" ...  Additionally, any registered accounts in the wallet (like a faucet account) will appear in the Registered Account list.

Checkboxes are now needed above in the 1st table.  This allows the user to check off which balances will be assigned to the selected registered account below.

**Comment by jcalfee** [comment]
Additionally, any registered accounts in the wallet (like a faucet account) will appear in the Registered Account list.

**Comment by bytemaster** [comment]
I am thinking that the table should not use center alignment. 

If the user selects an account below, it should automatically check all matching balances above. 

Make sure the user interface can handle a wallet with 1000+ balances, in other words the top table needs to be scrollable.

**Comment by svk31** [comment]
I assume you want users to be able to assign any balance to any of their accounts? I have a hard time seeing how this layout handles that, how about adding a fourth column to the table containing a dropdown where you can choose where to assign the balances?

This dropdown would be filtered of course as you say, and could be auto-initialized to the account of the same name if the user has the private keys to that account. 

If you absolutely don't want any accounts selected by default we should at least provide a button that will fill all fields, because I for one do not want to manually select 1000+ accounts!

**Comment by xeroc** [comment]
What will happen if my wallet only contains `active_keys` but not `owner_keys`?
